Gopkg.toml

```diff
@@ -24,7 +24,7 @@
 
 [[constraint]]
   name = "github.com/kubernetes-csi/external-snapshotter"
-  version = "~v1.1.0"
+  version = "v2.0.0-rc4"
```

Comment:
:) I guess we're still waiting on the GA release? This is fine for now.

Reply:
Yeah, it appears so. @xing-yang do you have a rough idea of when 2.0's release will happen?

Reply:
We still have a few PRs that need to be merged in external-snapshotter.  I'd like to get them all in by the end of this week, but I suspect it could be dragged a little longer.  Due to the holidays, we may have to wait until the first week of the new year.  Will let you know once that happens.

By the way, the snapshot APIs will not change between v2.0.0 and v2.0.0-rc4.  We are updating the controller logic.

Reply:
@nrb @skriss Snapshot v2.0.0 is released: https://github.com/kubernetes-csi/external-snapshotter/releases/tag/v2.0.0

Reply:
Thanks Xing, I'll update this repo to 2.0.0